.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,57 +1,131 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you:
+<!--  
+Thanks for submitting a pull request!
 
-1. If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
-2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
-https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
-3. Ensure you have added or ran the appropriate tests for your PR: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
-4. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
-5. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
-6. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+If this is your first time submitting a PR, please familiarise yourself with:
+
+  Contributor Guidelines 
+  https://git.k8s.io/community/contributors/guide#your-first-contribution
+
+  Developer Guide 
+  https://git.k8s.io/community/contributors/devel/development.md#development-guide
+
+The following are additional useful links for PRs:
+
+  Best Practices for Faster Reviews 
+  https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
+
+  Marking Unfinished PRs for Review or as a Work in Progress (WIP)
+  https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
+-->
+
+**Which is the sponsoring or primary SIG associated with this PR?**
+<!--
+Label the sponsoring or primary SIG associated with this PR.
+
+For reference on available SIGs, you can find more details at:
+https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md#labels-that-apply-to-all-repos-for-both-issues-and-prs
+
+Add only ONE sig here via a /sig-primary <name> entry.
 -->
+/sig-primary ???
+
+> DISCUSSION REQUIRED
+>
+> Should we use the above pattern and update the label prow plugin ? https://github.com/kubernetes/test-infra/tree/master/prow/plugins/label. Logic would have to ensure that there is only a single primary sig associated. This metadata can be leveraged by other processes outside of the release notes.
+> 
+> or, if this is ONLY relevant to release notes then, we can add here as a `` ```primary-sig`` code block.
 
 **What type of PR is this?**
-> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
```

Comment:
I would prefer to keep the "uncomment" format since it's easier than typing it out completely.

Reply:
>  Can a PR specify multiple "kinds"?

Generally, most PRs have a single "kind" but it is possible to have multiple "kinds" too -- a PR can have `kind/feature` and `kind/api-change`.

> If so, should I remove the language around only selecting one?

No strong opinions on this one. I think it should be fine to remove the language around selecting one, but a thing to note here is that we need _at least one_ "kind" or else the PR will get the `needs-kind` label and won't get merged. :grimacing:

Reply:
@nikhita I think a single kind is fine, we were primarily hoping for a `kind/deprecation` label, maybe a `kind/metrics-change` label but that may be confusing with `api-change` already existing. . .
We have changed the [release notes template](https://github.com/kubernetes/sig-release/edit/master/release-team/role-handbooks/release-notes/relnotes-template.md) significantly over the past  couple of releases with `deprecations` and `metrics` as well as  `urgent upgrade notes` being added. Since this is becoming the "way we do relnotes by hand" some tooling to support further automation would prevent a scramble just prior to the release and reduce the strain on the relnotes team/contributors who consistently help out :smile:

Reply:
@nikhita The use of blockquote here and the requirement for having to hit enter and remove leading whitespace felt complicated. I wish markdown had a line comment character that we could use here.

I have updated the PR by using the `~~` strikethrough wrapping characters. It is less visually busy in the rendered version. Thoughts? At this point, after playing with a number of options to improve this, I'm at a loss. So if the strikeththrough option is not appealing to everyone, I'll move it back to blockquotes.